test/06_docker_images/cibuildwheel_test.py

```diff
@@ -8,7 +9,8 @@
 def test():
     project_dir = os.path.dirname(__file__)
 
-    if utils.platform != 'linux':
+    if utils.platform != 'linux' or platform.machine() not in ['x86_64',
+                                                               'i686']:
```

Comment:
Perhaps this is cleaner:

```python
    if utils.platform != 'linux':
        pytest.skip('this test is only relevant to the linux build')
    if platform.machine() not in ['x86_64', 'i686']:
        pytest.skip('this test is currently only possible on x86_86/i686 due to availability of alternative images')
```

Reply:
Done in: https://github.com/joerick/cibuildwheel/pull/273/commits/05df9bbfd413122bf6fbda0e317abfe882290416